code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -0,0 +1,91 @@
+// Uniform slot
+/datum/gear/uniform
+	subtype_path = /datum/gear/uniform
+	slot = slot_w_uniform
+	sort_category = "Uniforms and Casual Dress"
+
+/datum/gear/uniform/skirt
+	display_name = "plaid skirt, blue"
+	path = /obj/item/clothing/under/dress/plaid_blue
+
+/datum/gear/uniform/skirt/purple
+	display_name = "plaid skirt, purple"
+	path = /obj/item/clothing/under/dress/plaid_purple
+
+/datum/gear/uniform/skirt/red
+	display_name = "plaid skirt, red"
+	path = /obj/item/clothing/under/dress/plaid_red
+
+/datum/gear/uniform/skirt/black
+	display_name = "skirt, black"
+	path = /obj/item/clothing/under/blackskirt
+
+/datum/gear/uniform/skirt/ce
+	display_name = "skirt, ce"
+	path = /obj/item/clothing/under/rank/chief_engineer/skirt
```

Comment:
Let me wear a dress as a CE; it totally won't get caught in the gears, equipment or anything like that; totes not a safety hazard.